### DISCUSSION TITLE
uri malformed error

### DISCUSSION THREAD

**Discussion by jhs1965net** [MuiseDestiny/zotero-gpt#184] [Q&A] [answered]
I write a wrong api and now I have an error every time I use gpt 

uri malformed error and if uninstall it and reinstall is the same ..

Thanks a lot

James

**Comment by MuiseDestiny** [comment, marked as answer]
you can type `/api your new api` and press enter set set a new api.

**Reply by MuiseDestiny** [reply]
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/51939531/da2d5c0b-a59c-4687-b4cd-c879cc92132c)

like this.

**Comment by jhs1965net** [comment]
Sorry , the Zotero API? With one and how , sorry , so nerd by my part

James

**Comment by jhs1965net** [comment]
Hi , it’s me again , hope you are doing ok and merry Christmas

Is it possible to fix : network error , wrong uri?

Kindly

James

**Reply by MuiseDestiny** [reply]
Thank you, merry Christmas. You can input /api default, and then press Enter. Your api will be right.

**Comment by jhs1965net** [comment]
You  are great , working thanks a lot.
Best wishes

James
